From 443e46d2d9d29cb9d20192e4978eb633bbbcc9b4 Mon Sep 17 00:00:00 2001
From: David Herrmann <dh.herrmann@gmail.com>
Date: Fri, 10 Aug 2018 16:35:31 +0200
Subject: [PATCH 2/2] bus-launch: use __linux__ over __linux

The canonical way to check for linux support is '__linux__', not
'__linux'. Particularly, on ppc64le '__linux' is not defined and the
build will fail.

For reference, see:

    https://sourceforge.net/p/predef/wiki/OperatingSystems/
---
 bus/at-spi-bus-launcher.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/bus/at-spi-bus-launcher.c b/bus/at-spi-bus-launcher.c
index 354c04eec7ba1a1ee034a245ee25c0139c48a1e5..b65301def88c2ff8aa5d10ada67507d0201b0984 100644
--- a/bus/at-spi-bus-launcher.c
+++ b/bus/at-spi-bus-launcher.c
@@ -23,11 +23,11 @@
 #include "config.h"
 
 #include <unistd.h>
 #include <string.h>
 #include <signal.h>
-#ifdef __linux
+#ifdef __linux__
 #include <sys/prctl.h>
 #include <sys/socket.h>
 #include <sys/un.h>
 #endif
 #include <sys/wait.h>
@@ -275,11 +275,11 @@ setup_bus_child_daemon (gpointer data)
   close (app->pipefd[0]);
   dup2 (app->pipefd[1], 3);
   close (app->pipefd[1]);
 
   /* On Linux, tell the bus process to exit if this process goes away */
-#ifdef __linux
+#ifdef __linux__
   prctl (PR_SET_PDEATHSIG, 15);
 #endif
 }
 
 static gboolean
-- 
2.19.0.rc0

